spryker/dockersuite_development.yml

```diff
@@ -229,3 +221,11 @@ sections:
             command: "vendor/bin/console frontend:zed:build"
             groups:
                 - zed
+
+    jenkins-up:
+        jenkins-generate:
+            command: "vendor/bin/console scheduler:setup"
+
+        jenkins-enable:
+            command: "vendor/bin/console scheduler:resume"
```

Comment:
are they in the correct order?

Reply:
Yep, Jenkins jobs enabled at the end of install flow